frameworks/digital-outcomes-and-specialists/questions/briefs/startDate.yml

```diff
@@ -2,18 +2,17 @@ name: Latest start date
 question: Latest start date
 question_advice: |
   Say when you need suppliers to start work. When you shortlist, you can exclude suppliers who can't start by this date.
-hint: 'eg 31/12/2020'
-type: text
+hint: 'eg 31 12 2020'
```

Comment:
I don't think we need to make any changes to the DOS1 question as this is no longer in use for editing.

Information will be played back (for example on published briefs) but I don't see any reason why we need to update it to a date question (even if it will fall back to string functionality)